ArduCopter/GCS_Mavlink.h

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#include "config.h"
```

Comment:
Heh, this was needed to fix compiler errors, but since the #if left the header file the compile error (undefined values for ADVANCED_FAILSAFE and ENABLE) doesn't happen anymore :) pushed the fix.